test/docker/docker-compose.yml

```diff
@@ -0,0 +1,97 @@
+version: "3"
+services:
```

Comment:
Do you mind adding a short comment or maybe even a small README about integration test setup and how do they work?

Reply:
ya good idea https://github.com/stellar/typescript-wallet-sdk/pull/74/commits/706c890ed3a1b5899d1a5ecdee3892c04930f664